### DISCUSSION TITLE
couldn't send discord message AbortController is not defined

### DISCUSSION THREAD

**Discussion by l0g4n1337** [jef/streetmerchant#3109] [Help]
[4:35:13 PM] info :: ✖ [gamestop-it] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[4:35:13 PM] info :: ✖ [unieuro] [sony (sonyps5de)] ps5 digital :: OUT OF STOCK
[4:35:14 PM] info :: ✖ [comet] [sony (sonyps5c)] ps5 console :: OUT OF STOCK
[4:35:15 PM] info :: 🚀🚨 [amazon-it] [sony (sonyps5de)] ps5 digital :: IN STOCK 🚨🚀
https://www.amazon.it/dp/B08KJF2D25
[4:35:15 PM] error :: ✖ couldn't send discord message AbortController is not defined {
  "name": "ReferenceError",
  "code": 500,
  "method": "post",
  "path": "/webhooks/1030445795062861864/SihgkcnVrnvGvZrCHG75yPJw3RTbpa-GvXmyLlv0F1rT4X2SHhun0q7gxWL-PYaohgNY?wait=true",
  "requestData": {
    "json": {
      "content": "1048717389275082864",
      "tts": false,
      "embeds": [
        {
          "title": "_**Stock alert!**_",
          "type": "rich",
          "description": "> provided by [streetmerchant](https://github.com/jef/streetmerchant) with :heart:",
          "url": null,
          "timestamp": "2022-12-13T15:35:15.329Z",
          "color": 5420936,
          "fields": [
            {
              "name": "Store",
              "value": "amazon-it",
              "inline": true
            },
            {
              "name": "Price",
              "value": "€38.69",
              "inline": true
            },
            {
              "name": "Product Page",
              "value": "https://www.amazon.it/dp/B08KJF2D25",
              "inline": false
            },
            {
              "name": "Add to Cart",
              "value": "https://www.amazon.it/gp/aws/cart/add.html?ASIN.1=B08KJF2D25&Quantity.1=1",
              "inline": false
            },
            {
              "name": "Brand",
              "value": "sony",
              "inline": true
            },
            {
              "name": "Model",
              "value": "ps5 digital",
              "inline": true
            },
            {
              "name": "Series",
              "value": "sonyps5de",
              "inline": true
            }
          ],
          "thumbnail": {
            "url": "https://raw.githubusercontent.com/jef/streetmerchant/main/docs/assets/images/streetmerchant-logo.png"
          },
          "image": null,
          "author": null,
          "footer": null
        }
      ],
      "username": "streetmerchant"
    },
    "files": []
  },
  "stack": "ReferenceError: AbortController is not defined\n    at RequestHandler.execute (C:\\Users\\luigi\\Desktop\\streetmerchant-main\\streetmerchant-main\\node_modules\\discord.js\\src\\rest\\RequestHandler.js:172:15)\n    at RequestHandler.execute (C:\\Users\\luigi\\Desktop\\streetmerchant-main\\streetmerchant-main\\node_modules\\discord.js\\src\\rest\\RequestHandler.js:176:19)\n    at RequestHandler.push (C:\\Users\\luigi\\Desktop\\streetmerchant-main\\streetmerchant-main\\node_modules\\discord.js\\src\\rest\\RequestHandler.js:50:25)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:95:5)\n    at async WebhookClient.send (C:\\Users\\luigi\\Desktop\\streetmerchant-main\\streetmerchant-main\\node_modules\\discord.js\\src\\structures\\Webhook.js:169:15)"
}

**Comment by Mr-Tech-13** [comment]
Make sure your dotenv is configured properly

**Reply by Mr-Tech-13** [reply]
If it is, try running npm i, then npm audit fix
